OpenRA.Mods.Common/Traits/Armament.cs

```diff
@@ -39,7 +39,8 @@ public class ArmamentInfo : PausableConditionalTraitInfo, Requires<AttackBaseInf
 		[Desc("Time (in frames) until the weapon can fire again.")]
 		public readonly int FireDelay = 0;
 
-		[Desc("Muzzle position relative to turret or body. (forward, right, up) triples")]
+		[Desc("Muzzle position relative to turret or body, (forward, right, up) triples.",
+			"If weapon Burst = 1, it cycles through all listed offsets, otherwise the offset corresponding to current burst is used.")]
```

Comment:
current shot in the burst, right?

Reply:
According to what I found online, 'burst' stands for a shot in a salvo.

Reply:
See #13090 in relation to this ongoing confusion.